neo4j/src/test/java/org/hibernate/ogm/datastore/neo4j/test/EmbeddedGraphDatabaseFactoryTest.java

```diff
@@ -68,6 +66,51 @@ public void testLoadMalformedPropertiesLocation() throws Exception {
 		factory.create().shutdown();
 	}
 
+	@Test
+	public void testInstantiationTwoFactoriesForOnePath() {
+		//create first factory
+		EmbeddedNeo4jGraphDatabaseFactory factory1 = new EmbeddedNeo4jGraphDatabaseFactory();
+		Properties properties = new Properties();
+		properties.put( Neo4jProperties.DATABASE_PATH, dbLocation );
+		properties.put( Neo4jProperties.CONFIGURATION_RESOURCE_NAME, neo4jPropertiesUrl().toExternalForm() );
+		factory1.initialize( properties );
+		GraphDatabaseService graphDatabaseService1 = factory1.create();
+
+		//create second factory
+		EmbeddedNeo4jGraphDatabaseFactory factory2 = new EmbeddedNeo4jGraphDatabaseFactory();
+		factory2.initialize( properties );
+		GraphDatabaseService graphDatabaseService2 = factory2.create();
```

Comment:
When you mention that it wasn't working with multiple path I thought you meant when using different folders. If you are using the same folder is supposed to give you errors because Neo4j acquire a lock on it and it won't let you change the database. It's not a use case we need to solve.